README.md

```diff
@@ -41,12 +41,31 @@ All headers are recorded as a JSON blob. Known sensitive headers are stripped fr
 | [`cf-connecting-ip`](https://developers.cloudflare.com/fundamentals/reference/http-request-headers/#cf-connecting-ip) | The client IP address connecting to Cloudflare to the origin web server. |
 | [`cf-ipcountry`](https://developers.cloudflare.com/fundamentals/reference/http-request-headers/#cf-ipcountry) | The two-character country code of the originating visitor’s country. |
 | `x-gateway-service-id` | The requested service's identifier. |
-| `x-gateway-service-name` | The requested service's human readable name. | 
+| `x-gateway-service-name` | The requested service's human readable name. |
 | `x-gateway-identifier-for-vendor` | An alphanumeric string that uniquely identifies a device to the connecting application’s vendor. |
 | `x-gateway-bundle-identifier` | The connecting application's bundle identifier. |
 | `x-gateway-bundle-version` | The connecting applications version number. |
 
 The following is also recorded:
 * Full URL of the request
-* Responses status code 
+* Responses status code
 * Any server specific errors
+
+## Data Retention
+
+All metadata is kept for a period of 90 days. Beyond this timeframe, the metadata undergoes permanent deletion with **no backup procedures in place.**
```

Comment:
Can we make this configurable?

Reply:
It is already customizable through the DELETE_OLD_DATA_BEFORE variable in the wrangler.toml file. Presently, it is configured to `-3 months`, but you have the flexibility to modify it as needed.